conu/backend/docker/image.py

```diff
@@ -384,6 +384,53 @@ def callback():
             container_name = actual_name
         return DockerContainer(self, container_id, popen_instance=popen_instance, name=container_name)
 
+    def run_via_api(self, container_params):
+        """
+        create a container using this image and run it in background via Docker-py API.
+        https://docker-py.readthedocs.io/en/stable/api.html
+        :param container_params: DockerContainerParameters
+        :return: instance of DockerContainer
+        """
+
```

Comment:
To be able to run container with `run_via_api()` without commands or parameters, please add
```
if not container_params:
     container_params = DockerContainerParameters()
```